webprojects/snippets.md

```diff
@@ -0,0 +1,107 @@
+* DON’T translate it, localize it! Usually the English copy is full of cultural references that may not apply to your local culture and it to the wrong audience. Try to capture the essence of the message, and make it relevant to your market.
+* DO read the comments for some messages that provide info on context, type of campaign, and length limitation of the message.
+* DO collaborate with other community members. You may come up with something better than working alone.
+* Do check for spelling and grammatical errors. Snippets have short life cycle. Get it right before it reaches millions of users.
+* DO NOT rush to complete the work and then never look at it again. Think through for few days before you consider it final.
+* Do check due date of the task. Snippets is high priority and time sensitive.
+
+## Where and when to test snippets?
+
+Currently, there is no staging server for localizers to preview the snippets or layout before they are scheduled to go live.
+
+Snippets usually go live as soon as a day after the snippets’ due date, or as late as a few weeks later because they are scheduled against other priority messages.
+
+To see snippets, you must set Firefox browser as the default browser. Then type `about:home`in the Awesome Bar, the current running snippets will be on display. A snippet can run between 4-7 days. Sometimes a set of snippets rotates within a given time frame; sometimes one snippet runs a few days before it is replaced by another one. It all depends.
+
+Take a peek of the [currently running and scheduled snippets](https://snippets.mozilla.com/). It shows the country, the message, and the schedule of a snippet.
+
+If you spot a mistake in a snippet, [file a bug to the team](https://bugzilla.mozilla.org/enter_bug.cgi?product=Snippets&component=Service) right away. Specify language, current content, suggested correction, severity of the problem or the reason it is incorrect. And preferably, attach a screenshot.
+
+## FAQ: process to make a special snippet request
+
+If your language is not listed above and you would like to have a snippet created for special occasion, such as new product or feature launch, a major holiday or campaign, please follow these steps.
+
+### How to create a good snippet?
+
+#### Leverage what is already written and localize it
+
+If creating your own is challenging, feel free to leverage what was created by the snippets team. Browse through [the snippets written this year so far](https://github.com/mozilla-l10n/engagement-l10n/tree/master/en-US/snippets/2017), you may find a message that resonates to your particular market, product and/or feature launch or a campaign.
+
+#### Create your own
+
+Writing snippets takes a lot of effort. Follow [the same guidelines](https://docs.google.com/presentation/d/1-rR32aMK60u6EvHvRCyINI0J_KH0_shbD_jbPqF_VUI/edit#slide=id.g1632693f0a_0_21) that the snippets team adhere to.
```

Comment:
Remember to check that documents are accessible to volunteers when sharing links ;) Using a private window is an easy way to test